python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -511,6 +511,13 @@ def update_stop_sequence(db_manager: DatabaseManager) -> None:
     """
     Update canonical_stop_sequence  and sync_stop_sequence from static_route_patterns
     """
+    partition_by = (
+        StaticRoutePatterns.static_version_key,
+        StaticRoutePatterns.direction_id,
+        sa.func.coalesce(
+            StaticTrips.branch_route_id, StaticTrips.trunk_route_id
+        ).label("route_id"),
+    )
```

Comment:
why doesn't this just get added to the `static_canon` select query?

Reply:
It was just an effort to not make the statement a nested mess

Reply:
i think its fine. i was just curious.